torch/_export/serde/serialize.py

```diff
@@ -1554,6 +1628,44 @@ def deserialize_sym_int(self, s: SymInt) -> Union[int, torch.SymInt]:
                 f"SymInt has invalid field type {s.type} with value {s.value}"
             )
 
+    def deserialize_sym_float(self, s: SymFloat) -> Union[float, torch.SymFloat]:
+        val = s.value
+        if s.type == "as_expr":
+            expr_str = val.expr_str
+            hint = val.hint.as_float if val.hint else None
+
+            if expr_str in self.symbol_name_to_symbol:
+                sym = self.symbol_name_to_symbol[expr_str]
+            else:
+                sym = sympy.sympify(
```

Comment:
same here for the string -> AST de/serialization issue.

Reply:
Could you patch also this to align to your new PR?